NetKAN/RemoteTechStockConfigs.netkan

```diff
@@ -0,0 +1,21 @@
+{
+    "spec_version" : 1,
+    "identifier"   : "RemoteTechStockConfigs",
+    "abstract"     : "Adds ground stations compatible with Remote Tech in the locations of the stock dishes",
+    "$kref"        : "#/ckan/github/woeller/RemoteTech-StockConfigs",
+    "$vref"        : "#/ckan/ksp-avc/RTstock.version",
```

Comment:
There's no .version file in the ZIP corresponding to this property. Should there be?